ansq/tcp/types/connection.py

```diff
@@ -24,6 +24,7 @@ def __init__(
         message_queue: asyncio.Queue = None,
         on_message: Callable = None,
         on_exception: Callable = None,
+        on_close: Callable = None,
```

Comment:
Let's start adding full specs for callback callables. I doubt we can't do better then a bare `Callable`.

Reply:
Addressed in 6302787966c086c83bbcea35983e31d632a1d969